migrations/2019-11-11-162609_drop_email_from_user/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE  users DROP COLUMN email CASCADE;
```

Comment:
This is super picky, but can you remove the extra space between `TABLE` and `users`?

Reply:
:)